instrumentation/nacos-client-2.0.0/javaagent/build.gradle.kts

```diff
@@ -0,0 +1,24 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+muzzle {
+  pass {
+    group.set("com.alibaba.nacos")
+    module.set("nacos-client")
+    versions.set("[2.0.0,)")
+    skip("0.5.0", "0.6.1", "1.1.2", "1.1.4", "1.4.7")
```

Comment:
you have the versions.set() to only include versions 2.0.0 and above of the nacos-client module, so I wouldn't expect to need to skip anything below 2.x ?